test/e2e/apimachinery/webhook.go

```diff
@@ -343,6 +347,40 @@ func testWebhook(f *framework.Framework) {
 		framework.Failf("expect error contains %q, got %q", expectedErrMsg, err.Error())
 	}
 
+	By("create a configmap that should be admitted by the webhook")
+	// Creating the configmap, the request should be admitted
+	configmap = &v1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: allowedConfigMapName,
+		},
+		Data: map[string]string{
+			"admit": "this",
+		},
+	}
+	_, err = client.CoreV1().ConfigMaps(f.Namespace.Name).Create(configmap)
+	Expect(err).NotTo(HaveOccurred())
+
+	By("update the admitted configmap to a non-compliant one should be rejected by the webhook")
+	// Update the configmap (PUT), the request should be rejected
+	toNonCompliantFn := func(cm *v1.ConfigMap) {
+		if cm.Data == nil {
+			cm.Data = map[string]string{}
+		}
+		cm.Data["webhook-e2e-test"] = "webhook-disallow"
+	}
+	_, err = updateConfigMap(client, f.Namespace.Name, allowedConfigMapName, toNonCompliantFn)
+	Expect(err).NotTo(BeNil())
+	if !strings.Contains(err.Error(), expectedErrMsg) {
+		framework.Failf("expect error contains %q, got %q", expectedErrMsg, err.Error())
+	}
+	// Update the configmap (PATCH), the request should be rejected
```

Comment:
Similar as above.  Can we add a By() call that expalins this step and also mentions that we are issuing a PATCH, then delete the comment?